packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -505,11 +505,6 @@ const PersonalDetailsForm = props => {
                                                 legal_entity_name: getLegalEntityName('maltainvest'),
                                             }
                                         )}
-                                        renderlabel={title => (
```

Comment:
is there a specific reason we're removing this?
Is it related to trading assessment?

Reply:
there was a warning in console regarding this prop. we are passing this prop to the html input, but it doesn’t accept such props